Source/ACE/Entity/Landblock.cs

```diff
@@ -172,7 +172,18 @@ public void AddWorldObject(WorldObject wo)
             {
                 // send them the initial burst of objects
                 Log($"blasting player \"{(wo as Player).Name}\" with {allObjects.Count} objects.");
-                Parallel.ForEach(allObjects, (o) => (wo as Player).TrackObject(o));
+                Parallel.ForEach(allObjects, (o) =>
+                {
+                    if (o.Guid.IsCreature())
+                    {
+                        if ((o as Creature).IsAlive)
+                            (wo as Player).TrackObject(o);

```

Comment:
smite the drudge in holtburg, logout, don't reset the server, log back in, crash the server.. didn't check into this further.